packages/openneuro-components/src/search-page/FacetBlockContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { FacetSelect } from '../facets/FacetSelect'
+import { FacetRadio } from '../facets/FacetRadio'
+import { modalities, showDatasetsRadio } from '../mock-content/facet-content'
+
+export const FacetBlockContainer = () => {
```

Comment:
This has context dependent children, so it will need to use `children` or render props too. Something like
```
const FacetBlockContainer = ({ children, renderMore }) => {
  const [showMore, setShowMore] = useState(false)
  const toggleShowMore = () => setShowMore(prevState => !prevState)
  return (
    <div>
      {children}
      <button
        onClick={toggleShowMore}
      >more+</button>
      {showMore && renderMore()}
    </div>
  )
}
```
which would be used like
```
< FacetBlockContainer
  renderMore={() => (
    <>
      <GenderButtons/>
      <PubDateRangePicker />
      <Etc />
    </>
  )}
>
  <FacetSelect
    selected={selected}
    ...
  />
  <FacetSelect ... />
  <FacetRadio ... />
</FacetBlockContainer >
```

Reply:
I used an Accordion Component for the More + functionality. Let me know if this isn't a good approach.

Reply:
I am removing the "more" accordion. Now that the facets are all accordions this isnt needed